freqtrade/utils.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from argparse import Namespace
+from typing import Any, Dict
+
+from freqtrade.configuration import Configuration
+from freqtrade.exchange import supported_exchanges
+from freqtrade.state import RunMode
+
+
+logger = logging.getLogger(__name__)
+
+
+def setup_configuration(args: Namespace, method: RunMode) -> Dict[str, Any]:
+    """
+    Prepare the configuration for the Hyperopt module
```

Comment:
Can this be reused for hyperopt ... ? 
anyway i don't think its necessary for this subcommand (see other comment)

Reply:
> Can this be reused for hyperopt ... ?
> anyway i don't think its necessary for this subcommand (see other comment)

Also thought about this. Even without being called from hyperopt, the next step of getting rid of get_market_pairs script will be adding another subcommand, `list-pairs`, as I said. This function will definitely be called from its implementation.